query-languages/m/table-fromlist.md

```diff
@@ -61,3 +61,29 @@ Table.FromRecords({
     [CustomerID = 2, Name = "Jim"]
 })
 ```
+
+## Example 3
```

Comment:
Thanks for suggesting this.

What if instead of the current example sequence, the first two examples showed the effect of different splitting options, and the third example was the Record.FieldValues one?

The first example could be:
Table.FromList({"a,apple", "b,ball", "c,cookie", "d,door"}, null, {"Letter", "Example Word"})

And the second could be:
Table.FromList({"a,apple", "b,ball", "c,cookie", "d,door"}, Splitter.SplitByNothing(), {"Letter and Example Word"})

Reply:
Thanks for the review. I applied the changes you suggested. I also added a link to the splitter functions.